zipline/finance/slippage.py

```diff
@@ -502,3 +502,30 @@ def get_simulated_impact(self,
     def get_txn_volume(self, data, order):
         volume = data.current(order.asset, 'volume')
         return volume * self.volume_limit
+
+
+class FixedBasisPointsSlippage(SlippageModel):
+    """
+    Model slippage as a fixed percentage of fill price.
```

Comment:
This should probably also mention that it executes the full order immediately.

Reply:
updated the docstring